src/Targets/Microsoft.CSharp.DesignTime.Default.props

```diff
@@ -0,0 +1,13 @@
+<!--
+
+  This file contains C# Visual Studio and designer-related properties and items for C# projects.
+  
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Optimize>false</Optimize>
```

Comment:
Hmm.. These aren't designtime value though. These properties get passed to the compiler. Should these values be in CSharp\VB targets instead?